### ISSUE TITLE
[Bug][a11y] The "password tips" when creating an account cannot be dismissed

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/fenix#16028]
## Steps to reproduce
1. Turn voice assistant or talkback on your device.
2. Go to Settings - Turn on sync and create a new account.

### Expected behavior
The user can easily create an account.

### Actual behavior
The user is having difficulties confirming the password; the password tips cannot be dismissed, blocking the confirmation of the password.
Workaround: complete the checkboxes below the password area, then return to the password confirmation.

### Device information
* Android device: Samsung Galaxy Tab S6 (Android 10)
* Fenix version: 10/19 Nightly, RC 82.1.0-rc.2, Beta 82.0.0-beta.6

<img src="https://user-images.githubusercontent.com/40694158/96467772-f819bb00-1233-11eb-9e0d-9b0f218e6829.png" width="300" height="550" />

**Comment by liuche** [comment]
Hey @jdragojevic do you know who handles the FxA creation page? It seems it's not accessible, and I'm wondering if that's something we should do in GV or if there's something the FxA team could audit.

**Comment by jdragojevic** [comment]
@liuche - which page?  if it's on accounts.firefox.com, it's an FxA wil clouser thing.

**Comment by liuche** [comment]
I filed this in the mozilla/fxa repo, see https://github.com/mozilla/fxa/issues/6797

**Comment by softvision-miralobontiu** [comment]
Reproducible on Beta 86.0.0-beta.5 with Xiaomi Mi 8 Lite (Android 9).

**Comment by VitalyVPinchuk** [comment]
> Reproducible on Beta 86.0.0-beta.5 with Xiaomi Mi 8 Lite (Android 9).

Hello @lobontiumira !
I can't reproduce it on the latest master branch neither using Pixel 4 API 29 emulator nor physical device Xiaomi Mi 9T Pro.
Once the password criteria is met, the tip dismisses in a second.

Could you please check again?

**Comment by lobontiumira** [comment]
Hi!
I retested on the latest Beta 88.0.0-beta.2 build with Xiaomi Mi 8 Lite (Android 9) and I was not able to reproduce it anymore.
I will close this issue.
Thank you, @VitalyVPinchuk
